components/Molecule/figure.html

```diff
@@ -13,12 +13,12 @@
 .figure figcaption {
     text-align: center;
 }
-.figure figcaption a{
+.figure figcaption a {
     text-decoration: none;
 }
 </style>
 
-
+<markup>
```

Comment:
The HTML in a component needs to have `<markup>` around it in order to work in the pattern library.